composer/utils/checkpoint.py

```diff
@@ -119,6 +121,8 @@ def load_checkpoint(
             restoring the associated state. (default: ``False``)
         strict_model_weights (bool, optional): Whether or not to force that the checkpointed weights must exactly
             match the model weights. (default: ``False``)
+        load_ignore_missing_checkpoint (bool, optional): Whether or not to ignore missing files and emit a warning or otherwise
```

Comment:
Would it work to catch the `FileNotFoundException` exception directly in the trainer? Would prefer not to emit a warning every time.